### PR TITLE
{phys}[foss/2021a] PRISMS-PF v2.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
* https://github.com/easybuilders/easybuild-easyblocks/pull/2748
* https://github.com/easybuilders/easybuild-easyconfigs/pull/15690 (for `p4est` dependency)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2748
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node3109.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/88c7f7d2c40cb1f5e4db9e79ec70cd19 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--include-easyblocks-from-pr 2748"

**Review by branfosj — APPROVED** [review]
lgtm

File: easybuild/easyconfigs/p/PRISMS-PF/PRISMS-PF-2.2-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ('GSL', '2.7'),
+    ('HDF5', '1.10.7'),
+    ('METIS', '5.1.0'),
+    ('netCDF', '4.8.0'),
+    ('p4est', '2.8'),
+    ('PETSc', '3.15.1'),
+    ('zlib', '1.2.11'),
+]
+
+local_dealii_configopts = "-DCMAKE_BUILD_TYPE=Release "
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/p/PRISMS-PF/PRISMS-PF-2.2-foss-2021a.eb]: fixed in 44e3b5d

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/395f08389d163425da2533f2c0d0b678 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**